airbyte-integrations/connectors/source-hubspot/source_hubspot/api.py

```diff
@@ -40,6 +40,18 @@
 # we got this when provided API Token has incorrect format
 CLOUDFLARE_ORIGIN_DNS_ERROR = 530
 
+VALID_JSON_SCHEMA_TYPES = {"string", "integer", "number", "boolean", "object", "array", "bit"}
+
+KNOWN_CONVERTIBLE_SCHEMA_TYPES = {
+    "bool": "boolean",
+    "enumeration": "string",
+    "date": "string",
+    "date-time": "string",
+    "datetime": "string",
+    "json": "object",
```

Comment:
I don't think its a good idea to treat unknown custom data as an object. 
a) It could be a string containing invalid json
b) If this an object then json schema is expected to enumerate  list of properties 
c) In case is destination is relational DB with normalization turned on it would create unknown amount of tables for nested structure. 

I think its better to treat it as a string.

Reply:
good catch

Reply:
Agree